.pre-commit-config.yaml

```diff
@@ -2,14 +2,14 @@ fail_fast: true
 
 repos:
 
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.1.0
-    hooks:
-      - id: ruff-format
-        args: [integration, reflex, tests]
-      - id: ruff
-        args: ["--fix", "--exit-non-zero-on-fix"]
-        exclude: '^integration/benchmarks/'
+  # - repo: https://github.com/charliermarsh/ruff-pre-commit
```

Comment:
This should not be commented out.

If you want to commit and skip pre-commit, use the `--no-verify` flag.

Note that you'll still need to pass the CI.